### ISSUE TITLE
ColumnsToRowsTransform wrapped for Python in DHC

### ISSUE THREAD

**Issue by margaretkennedy** [deephaven/deephaven-core#3752]
This method is Javadoced, but we'd like it available for DHC Python users.

`_JColumnsToRowsTransform = jpy.get_type("io.deephaven.engine.table.impl.util.ColumnsToRowsTransform")
transformed_t = Table(_JColumnsToRowsTransform.columnsToRows(t.j_table, ...))`

**Comment by jmao-denver** [comment]
@chipkent This is the Java class to be wrapped . https://github.com/deephaven/deephaven-core/blob/main/engine/table/src/main/java/io/deephaven/engine/table/impl/util/ColumnsToRowsTransform.java

This is certainly not what I expect for a 'transpose' operation based on my understanding of the widely accepted definition, however I don't doubt its usefulness and surely recognize that it works on ticking tables where the canonical transpose doesn't make sense. 

Could you share your thoughts on whether we should do it now or punt it to sometime in the future?

**Comment by chipkent** [comment]
This looks more similar to `padas.wide_to_long` than to a transpose.  Are they the same, or is `ColumnsToRowsTransformation` different?
https://pandas.pydata.org/docs/reference/api/pandas.wide_to_long.html

**Comment by jmao-denver** [comment]
> This looks more similar to `padas.wide_to_long` than to a transpose. Are they the same, or is `ColumnsToRowsTransformation` different? https://pandas.pydata.org/docs/reference/api/pandas.wide_to_long.html

yes, they are very similar but for a non-data-science guy like me, neither is very easily understandable.
